streaming/text/convert/laion/laion400m/README.md

```diff
@@ -0,0 +1,37 @@
+## Instructions:
+
+### 1. Setup.
+
+```
+pip3 install img2dataset==1.41.0
+
+apt install bwm-ng htop iotop
```

Comment:
looks like this is only supported for ubuntu ? Maybe worth adding an instruction ?

Reply:
done